src/sentry/interfaces/user.py

```diff
@@ -78,6 +79,12 @@ def to_python(cls, data):
         except ValueError:
             raise InterfaceValidationError("Invalid value for 'ip_address'")
 
+        geo = data.pop('geo', None)
+        if not geo and ip_address:
+            geo = Geo.from_ip_address(ip_address)
+        elif geo and not isinstance(geo, Geo):
```

Comment:
I'm trying to debug why the `not isinstance(geo, Geo)` is needed.

For some reason an existing Geo instance can come through a UserInterface in `load-mocks`... it might be specific to `load-mocks` but I obviously don't want to break that script either.